### DISCUSSION TITLE
zls opens quickfix window on save if there are errors

### DISCUSSION THREAD

**Discussion by ghost** [zigtools/zls#1457] [Q&A] [answered]
I'm looking to disable this functionality, but I'm not sure where it's coming from. I tried overriding LSP handlers to no avail, and the lsp logs don't mention anything relatable (to my knowledge). Has anyone had the same issue and managed to disable it?

**Comment by llogick** [comment, marked as answer]
Hi,

Are you using (Neo)vim? This sounds similar to
* https://github.com/zigtools/zls/issues/856

**Reply by ghost** [reply]
Much appreciated, that indeed solved it. Thanks for bringing it to my attention!

**Reply by pseudocc** [reply]
TL;DR
```lua
vim.g.zig_fmt_parse_errors = 0
```
